zhaquirks/terncy/cl001.py

```diff
@@ -0,0 +1,88 @@
+"""Quirk for Xiaoyan CL001 ceiling light."""
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class CustomColorCluster(CustomCluster, Color):
```

Comment:
```suggestion
class ColorClusterTerncy(CustomCluster, Color):
```
(also apply suggestion to change cluster class name in `replacement` if applying this)